src/enrichment/lsst.rs

```diff
@@ -68,12 +69,99 @@ pub fn create_lsst_alert_pipeline() -> Vec<Document> {
     ]
 }
 
+pub struct Babamul {
+    kafka_producer: rdkafka::producer::FutureProducer,
+}
+
+impl Babamul {
+    fn new(config: config::Config) -> Self {
+        // Read Kafka producer config from kafka: producer in the config
+        // TODO: Do this properly
+        let kafka_producer_host = config
+            .get_string("kafka.producer")
```

Comment:
seems decoupled from the internal producer env var I saw earlier, shouldn't both read that value from the same place?